src/index.html

```diff
@@ -5,9 +5,60 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card" data-qa="card">
+
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+      >
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br> 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__review">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price price">
+        <div class="price__text">Price:</div>
+
+        <div class="price__value">$2,199</div>
+      </div>
+
+      <a
+      href="#"
+      class="card__button"
+      data-qa="hover"
+      >
```

Comment:
```suggestion
      <a
        href="#"
        class="card__button"
        data-qa="hover"
      >
```

There should be 2-space indentation for attributes